File: server/src/main/java/com/example/schoolsite/workWithDatabase/repo/ClassroomRepository.java
package com.example.schoolsite.workWithDatabase.repo;

import com.example.schoolsite.entity.Classroom;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ClassroomRepository extends JpaRepository<Classroom, Long> {
    Classroom findClassroomByName(String name);
}

File: server/src/main/java/com/example/schoolsite/workWithDatabase/repo/ClassroomRepository.kt
package com.example.schoolsite.workWithDatabase.repo

import com.example.schoolsite.entity.Classroom
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ClassroomRepository : JpaRepository<Classroom?, Long?> {
    fun findClassroomByName(name: String?): Classroom?
}